api/pom.xml

```diff
@@ -40,6 +40,11 @@
     </parent>
 
     <dependencies>
+        <dependency>
+            <groupId>com.conversantmedia</groupId>
```

Comment:
Are there other queues available, please put examples of other queues? Though this one solves our current problem but the CPU usage is too high.